website/content/docs/external_plugins/_index.md

```diff
@@ -210,6 +210,8 @@ Below is an example that includes pre-fetched method results for a static direct
 
 **NOTE:** Only implement `metadata` if there is additional information about your entry that is not provided by the `meta` attribute.
 
+**NOTE:** If you have some background in Ruby, then check out the [wash](https://github.com/puppetlabs/wash-ruby) gem.
```

Comment:
I would say: `Check out the [wash](https://github.com/puppetlabs/wash-ruby) gem. It provides a framework for implementing external plugins in Ruby.` That project should probably provide a badge linking to rubygems.org/gems/wash.

Reply:
Updated.